shared/src/output.rs

```diff
@@ -0,0 +1,365 @@
+use libherokubuildpack::command::CommandExt;
+use libherokubuildpack::write::line_mapped;
+use std::process::{Command, Output};
+use std::time::{Duration, Instant};
+
+pub fn print_buildpack_name(buildpack_name: impl AsRef<str>) {
+    let buildpack_name = buildpack_name.as_ref();
+    println!("{ANSI_BUILDPACK_NAME_CODE}# {buildpack_name}{ANSI_RESET_CODE}\n");
+}
+
+pub fn print_section(text: impl Into<BuildpackOutputText>) {
+    let text = text.into().to_ansi_string();
+    println!("{ANSI_RESET_CODE}- {text}");
```

Comment:
All code clearing needs to happen before the newline of the last print rather than at the beginning of the next line. Otherwise we end up with red/yellow/blue `remote:`-s.

Reply:
It won't hurt anything, I was just a little thrown off. 

I think what I really want is the output to look at. I said so earlier as a non-blocker, but I'll upgrade that to a strong request. I think that's more valuable than me making random comments about your comments i.e. I care more about the output than the implementation and need a way to view/iterate on it.